Dockerfile

```diff
@@ -13,6 +14,6 @@ RUN poetry config virtualenvs.create false \
 COPY airbyte_cdk ./airbyte_cdk
 
 # Build and install the package
-RUN poetry build && pip install dist/*.whl
+RUN pip install dist/*.whl
```

Comment:
Using poetry build here would overwrite the CDK version obtained from the build artifact